test/services/DataStoreServiceSpec.scala

```diff
@@ -42,6 +45,22 @@ class DataStoreServiceSpec extends SpecBase {
       }
     }
 
+    "return unverified email" in new Setup {
+      val undeliverableEventData = UndeliverableInformationEvent("someid", "somevent", "someemail", "", Some(100), Some("sample"), "sample")
```

Comment:
Could you please split the line over multiple line as it has become long line(more than 120 characters). Same for line 52?

If it is being shown properly on your machine then ignore this comment